OS-SPEC.md

```diff
@@ -7,12 +7,23 @@ This document attempts to define a minimal set of things that must exist for mos
 
 ACIs that define the label `os=linux` can expect this environment by default.
 
+## Execution Environment
+
+It is RECOMMENDED that the following environment variables be set for each application's main process and any lifecycle processes:
```

Comment:
I was anticipating that there is still a nontrivial amount of software that does depend on these things and so it would be worth gently nudging implementers in that direction. But I don't feel strongly enough about it to keep fighting this :-)